policyengine_us/parameters/gov/states/ky/tax/income/exclusions/pension_income/other_retirement_income_sources.yaml

```diff
@@ -0,0 +1,10 @@
+description: Other retirement income for Part II of Kentucky Schedule P pension income exclusion.
+metadata:
+  unit: variable
+  label: KY Other Retirement Income Sources
+  reference:
```

Comment:
can you add a legal reference, or if it doesn't exist leave a comment noting that? same for other parameters